sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -422,6 +422,7 @@ export interface SessionReceiver<ReceivedMessageT extends ReceivedMessage | Rece
 
 // @public
 export interface SessionReceiverOptions {
+    abortSignal?: AbortSignalLike;
```

Comment:
Should it extend `OperationOptionsBase` instead?

Reply:
Makes sense!